src/components/dialogs/InvalidIncorporationApplicationDialog.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <v-dialog v-model="dialog" width="45rem" persistent :attach="attach"
+            content-class="invalid-incorporation-application-dialog">
+    <v-card>
+      <v-card-title id="dialog-title">Incorporation Application Does Not Exist</v-card-title>
+
+      <v-card-text>
+        <p class="genErr">The incorporation application you are attempting to open does not exist -
```

Comment:
Scott asked for new text in a comment in the ticket.

This will need a new title too... ask him or propose one if you can't get a hold of him.